opendrift/models/basemodel/simulation.py

```diff
@@ -65,6 +66,7 @@ def __init__(self,
                  export_buffer_length=100,
                  stop_on_error=False):
 
+        self.init = init  # Copying init to Simulation, is this ok?
```

Comment:
As long as possible I would prefer to avoid this, and rather copy the relevant data over. Also there is `init = init.copy()` further down to make sure the original init object is not mutated. If this is only necessary for `origin_marker` now we should rather copy that parameter. And if we have to in the future, we also store a *copy* of init.